.env.sample

```diff
@@ -1,2 +0,0 @@
-WEB3_INFURA_PROJECT_ID=XXX
-WALLET_MNEMONIC='XXX XXX'
```

Comment:
@JonathanStefanov let's keep this file in the project :)

Reply:
Ooh my bad. I'll change it to the updated sample as well on the next push